src/models/integration-test-scenario.ts

```diff
@@ -2,7 +2,7 @@ import { K8sModelCommon, K8sGroupVersionKind } from '../types/k8s';
 
 export const IntegrationTestScenarioModel: K8sModelCommon = {
   apiGroup: 'appstudio.redhat.com',
-  apiVersion: 'v1beta1',
+  apiVersion: 'v1beta2',
```

Comment:
There was some issue when trying to edit a "integration test scenario". The integration team said we should start using `v1beta2` api version.

Reply:
in my understanding, it should not - I tested with some older integration tests, and it's working fine